core/tests/karma.conf.js

```diff
@@ -94,16 +98,21 @@ module.exports = function(config) {
       }
     },
     plugins: [
+      'karma-jasmine-jquery',
```

Comment:
Why do we need this? It doesn't seem like anything extra is being done in Karma tests.